packages/react/src/blocks/Image.tsx

```diff
@@ -193,7 +193,7 @@ class ImageComponent extends React.Component<any, { imageLoaded: boolean; load:
         this.setState({
           load: true,
         });
-      } else if (typeof IntersectionObserver === 'function' && this.pictureRef) {
+      } else if (this.props.useIO && typeof IntersectionObserver === 'function' && this.pictureRef) {
```

Comment:
Can we make the prop name more clear? 
```suggestion
      } else if (this.props.useIntersectionObserver && typeof IntersectionObserver === 'function' && this.pictureRef) {
```